cjrace/almostyellow
# Add chopinliszt table

## Overview of changes

This connects up the shopping list in the admin side with a new table in the database.

Included notes on how to seed the database table if needed.

Also:

1. Adds clsx package for conditional applying of classes, and uuid package for generating uuids
2. Rename `lint-style` shortcut to `pretty-lint` as that's what I kept typing instead
3. Make the homepage title have a yellow to orange gradient
4. Stop the decision maker text area from jumping on render (by fixing a minimum height)
5. Reworked the header and colour scheme toggles, to simplify the code a little and ignore hydration errors when changing page after changing the colour scheme (after some playing around it seemed the best option for now)

## Checklist

- [x] I have tested these changes locally using `pnpm test`
- [x] I have added or updated tests for these changes (if applicable)

## Reviewer instructions

Main thing is to test that you can do all the things on the Chopin Liszt:
- Create new items
- Check list persists after refreshing the page
- Update the checkboxes, again refresh page to check
- Delete items, check they're still deleted after refreshing 

Also work checking changing the colour scheme and then navigating around different pages.
